azure-quantum/azure/quantum/optimization/problem.py

```diff
@@ -124,12 +127,53 @@ def add_term(self, c: Union[int, float], indices: List[int]):
         self.terms.append(Term(indices=indices, c=c))
         self.uploaded_blob_uri = None
 
-    def add_terms(self, terms: List[Term]):
-        """Adds a list of terms to the `Problem` representation
+    def add_terms(self, terms: List[Term],
+                  type: Union[Optional[GroupType], str] = None, c: Union[int, float] = 1):
```

Comment:
in my opinion we have 2 ways of handling this enum if we want to keep the "na" group type. We can remove the Optional, and set the default value to GroupType.combination. 
If we want to keep the None, then I think we get rid of the GroupType.combination entirely. I don't see where the requirement is to support the existing monomials but as a nested group.

Reply:
Implemented the first option